install/kubernetes/values.yaml

```diff
@@ -148,6 +148,8 @@ tetragon:
   enablePolicyFilterDebug: false
   # Enable latency monitoring in message handling
   enableMsgHandlingLatency: false
+tetragonPod:
```

Comment:
let's do 
```suggestion
podWatcher:
```
instead. i think this is more descriptive in terms of what this feature is about.

Reply:
sure.

Reply:
done